scripts/extract-and-replace-translations.ts

```diff
@@ -0,0 +1,286 @@
+/**
+ This script runs over all the code and looks for either:
+<Trans>Aanvraagformulier</Trans>
+or
+t('Aanvraagformulier')
+
+and replaces them with:
+<Trans i18nKey="authentication/views/registration-flow/r-4-manual-registration___aanvraagformulier">Aanvraagformulier</Trans>
+or
+t('authentication/views/registration-flow/r-4-manual-registration___aanvraagformulier')
+
+
+and it also outputs a json file with the translatable strings:
+{
+"authentication/views/registration-flow/r-4-manual-registration___aanvraagformulier": "Aanvraagformulier"
+}
+
+Every time the `npm run extract-translations` command is run, it will extract new translations that it finds
+(without i18nKey or not containing "___")
+and add them to the json file without overwriting the existing strings.
+
+We can now give the src/shared/translations/nl.json file to claire to enter the final copy.
```

Comment:
Wie is Claire? Is dit nog relevant in dit project of is dit iets AVO specifiek?

Reply:
Nee klopt, shameless copy paste van AvO 😄 ik zal dat generiek naar team meemoo richten